### DISCUSSION TITLE
How do I set Configuration in 3.x?

### DISCUSSION THREAD

**Discussion by corysimmons** [Norserium/react-indiana-drag-scroll#72] [Q&A] [answered]
I see these config options, but... where do I set them exactly? 🤔 

https://norserium.github.io/react-indiana-drag-scroll/docs/api/scroll-container#configuration

**Comment by Norserium** [comment, marked as answer]
@corysimmons, I assume that I should add the example in the documentation.

```tsx
<ScrollContainer
   mouseScroll={{
      inertia: false
   }}
 />
```

By the way, what do you want to configure?

**Reply by corysimmons** [reply]
![image](https://user-images.githubusercontent.com/224636/194112350-937bdf9e-37e8-436a-a6bb-09582a4aec3b.png)

🤦 
Sorry, I completely didn't notice.

I was trying to disable the rubberband effect. Thank you!

**Reply by Norserium** [reply]
@corysimmons, you are welcome!
